pages/en/lb3/Embedded-models-and-relations.md

```diff
@@ -135,12 +137,10 @@ For example, a Customer can have multiple email addresses and each email address
 ```
 
 {% include important.html content="
-
 Treat `embedsMany` as an actual relation, no different from `hasMany`, for example.
 This means that you cannot just POST the full object with embedded/nested data to create everything all at once.
 So using the example above to add a Customer and multiple email addresses would require two POST operations,
 one for the Customer record and one for the multiple email address data.
```

Comment:
I've validated that the embedMany does not have the above behaviour and the above needed to be removed. I've closed my task as this is being removed under this task. 

Validation: [strongloop-internal/scrum-apex#123](https://github.com/strongloop-internal/scrum-apex/issues/123#issuecomment-311979892)